registry/remote/retry/policy.go

```diff
@@ -88,15 +88,15 @@ type Backoff func(attempt int, resp *http.Response) time.Duration
 // jitter. The backoff is calculated as:
 //
 //	temp = backoff * factor ^ attempt
-//	interval = temp * (1 - jitter) + rand.Int63n(2 * jitter * temp)
+//	interval = temp * (1 - jitter) + rand.Int64N(2 * jitter * temp)
 //
 // The HTTP response is checked for a Retry-After header. If it is present, the
 // value is used as the backoff duration.
 func ExponentialBackoff(backoff time.Duration, factor, jitter float64) Backoff {
 	return func(attempt int, resp *http.Response) time.Duration {
-		var h maphash.Hash
-		h.SetSeed(maphash.MakeSeed())
-		rand := rand.New(rand.NewSource(int64(h.Sum64())))
+		hashSeed := seedFromMapHash()
+		timeSeed := uint64(time.Now().UnixNano())
+		rand := rand.New(rand.NewPCG(hashSeed, timeSeed))
```

Comment:
What's the advantage of moving to PCG for the randomness of the exponential backoff scenario?

Reply:
It's new and seems not harmful?